bootstrap.go

```diff
@@ -523,6 +538,26 @@ func upgradeToVer14(s Session) {
 	}
 }
 
+func upgradeToVer15(s Session) {
+	var err error
+	_, err = s.Execute(CreateGCDeleteRangeTable)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = s.NewTxn()
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = ddl.LoadPendingBgJobsIntoDeleteTable(s)
```

Comment:
This function is not reenterable, so you should update version and commit it in one transaction.